extensions/shared/jetpack-likes-and-sharing-panel.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { createSlotFill, PanelBody } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from '../utils/i18n';
```

Comment:
The wrapper is no more: #11743

We'll need to update this and merge with external deps above:

```suggestion
import { __ } from '@wordpress/i18n';
```